unity/ProjectSettings/TagManager.asset

```diff
@@ -50,6 +50,13 @@ TagManager:
   - WaterPlane
   - WaveSource
   - GlobalEntity
+  - Calculator
```

Comment:
Do we need all of these tags, when we use FindObjectOfType instead of FindObjectWithTag?